pystarport/pystarport/cluster.py

```diff
@@ -973,6 +973,36 @@ def query_tally(self, proposal_id):
             )
         )
 
+    def ibc_transfer(
+        self,
+        from_,
+        to,
+        amount,
+        channel,
+        version,
+        i=0,
+    ):
+        return json.loads(
+            self.raw(
+                "tx",
+                "ibc-transfer",
+                "transfer",
+                "transfer",  # src port
+                channel,
+                to,
+                amount,
+                "-y",
+                "--absolute-timeouts",  # FIXME avoid an ClientState decoding error
```

Comment:
I guess it needs fixing in SDK module?

Reply:
probably, that needs further investigation.

Reply:
https://github.com/cosmos/cosmos-sdk/issues/8059
submitted bug report